app/retail/views.py

```diff
@@ -206,6 +206,9 @@ def index(request):
         'know_us': know_us,
         'press': press,
         'articles': articles,
+        'hide_newsletter_caption': True,
```

Comment:
what is this 2 for?

Reply:
newletter use those 2 to show more label on the newsletter
If we are not using it -> we can remove it

Reply:
Was this tested? Why make these changes to the newsletter?